config/settings/segmentedtickerplant.q

```diff
@@ -6,9 +6,14 @@ multilog:`tabperiod;            // [tabperiod|none|periodic|tabular|custom]
 multilogperiod:0D01;            // Length of period for STP periodic logging modes
 errmode:1b;                     // Enable error mode for STP
 batchmode:`defaultbatch;        // [memorybatch|defaultbatch|immediate]
-replayperiod:`day               // [period|day|prior]
+replayperiod:`day               // [period|day|prior|tailer]
 customcsv:hsym first .proc.getconfigfile["stpcustom.csv"];       // Location for custom logging mode csv
 
+\d .ds
+segmentconfig:`segmenting.csv   // default segmenting config file
+filtermap:`filtermap.csv        // default filter mapping config file
```

Comment:
should this be replaced with striping.json?

Reply:
Yes, will remove this now

Reply:
These were just redundant lines from a previous version